tests/unit/models/test_redditors.py

```diff
@@ -1,5 +1,7 @@
 """Test praw.models.redditors."""
 
+from unittest import mock
```

Comment:
We use the mock that is found in the `mock` package, please import mock directly (`import mock`).

Reply:
Sure thing.